wrappers/python/simtk/openmm/app/internal/pdbstructure.py

```diff
@@ -795,11 +798,14 @@ def __init__(self, pdb_line, pdbstructure=None):
         try: self.formal_charge = int(pdb_line[78:80])
         except ValueError: self.formal_charge = None
         # figure out atom element
-        try:
-            # Try to find a sensible element symbol from columns 76-77
-            self.element = element.get_by_symbol(self.element_symbol)
-        except KeyError:
-            self.element = None
+        if self.element_symbol == extraParticleIdentifier:
+            self.element = 'EP'
```

Comment:
I'm not sure this is worthwhile anymore - because all the name replacement code (https://github.com/pandegroup/openmm/blob/master/wrappers/python/simtk/openmm/app/pdbfile.py#L97 and so on) will also need to move to PDBStructure. Worth doing this and taking the risk of breaking something or should we just leave it as is?